UCSCore/pom.xml

```diff
@@ -0,0 +1,209 @@
+<!-- Copyright 2018 IIT-CNR Licensed under the Apache License, Version 2.0 
+	(the "License"); you may not use this file except in compliance with the 
+	License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 
+	Unless required by applicable law or agreed to in writing, software distributed 
+	under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES 
+	OR CONDITIONS OF ANY KIND, either express or implied. See the License for 
+	the specific language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>it.cnr.iit.ucscore</groupId>
+	<artifactId>UCSCore</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.6.RELEASE</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+
+	<dependencies>
+		<dependency>
+			<groupId>it.cnr.iit.ucs</groupId>
+			<artifactId>UCSLibraries</artifactId>
+			<version>0.0.1-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>it.cnr.iit.ucs</groupId>
+			<artifactId>PolicyDecisionPoint</artifactId>
+			<version>1.0-SNAPSHOT</version>
```

Comment:
I think we should always stick to 3 digits ,e.g. 0.0.1 or 1.0.0
May be for maven central repo we can adhere to it

Reply:
I agree, I've already put some projects on three digits. but most of them have two.
Should I do this in the current PR?